docs/conf.py

```diff
@@ -1,205 +1,36 @@
-# -*- coding: utf-8 -*-
-#
 # Configuration file for the Sphinx documentation builder.
 #
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# http://www.sphinx-doc.org/en/master/config
-
-# -- Path setup --------------------------------------------------------------
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-import os
-import sys
-
-sys.path.insert(0, os.path.abspath(".."))
-sys.path.append(os.path.abspath('extensions'))
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
 # -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+import pathlib
+import sys
 
-project = "coc"
-copyright = "2019, mathsman5133"
-author = "mathsman5133"
-
-# The short X.Y version
-version = ""
-# The full version, including alpha/beta/rc tags
-release = ""
+root_project = pathlib.Path(__file__).parents[1].resolve()
+sys.path.insert(0, root_project.as_posix())
+sys.path.append((root_project / "docs").as_posix())
 
+project = 'coc'
+copyright = '2022, mathsman'
```

Comment:
This may be pedantic, but I'd prefer mathsman5133, only because that's my github name :) 
```suggestion
copyright = '2022, mathsman5133'
```